infrastructure/kube/keep-prd/btc-difficulty/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+resources:
+  - maintainer-statefulset.yaml
+
+namespace: default
+
+commonLabels:
+  app: maintainer
+  type: btc-difficulty
+  network: mainnet
+
+configMapGenerator:
+  - name: btc-difficulty-maintainer-accounts-info
+    files:
+      - config/btc-difficulty-maintainer-keyfile
+  - name: electrum-api-mainnet
+    literals:
+      - electrumx-url-tcp=electrum.blockstream.info:50001
+
+secretGenerator:
+  - name: btc-difficulty-maintainer-eth-accounts-password
+    files:
+      - .secret/eth-account-password
```

Comment:
The person deploying the maintainer on production will have to make sure they have the password for the maintainer account (`0xCb6Ed7E78d27FDff28127F9CbD61d861F09a2324`) stored in `.secret/eth-account-password`.